x-pack/solutions/security/plugins/security_solution/public/notes/pages/note_management_page.tsx

```diff
@@ -125,15 +126,27 @@ export const NoteManagementPage = () => {
   const notesSearch = useSelector(selectNotesTableSearch);
   const notesCreatedByFilter = useSelector(selectNotesTableCreatedByFilter);
   const notesAssociatedFilter = useSelector(selectNotesTableAssociatedFilter);
+  const { data: suggestedUsers } = useSuggestUsers({ searchTerm: '' });
   const pendingDeleteIds = useSelector(selectNotesTablePendingDeleteIds);
+  // This is a workaround for the server-side 'text' field type which tokenizes values,
```

Comment:
Absolutely not important, but if you're making changes to the PR for the license above, I would move this block of code by down where it was originally (below the ` const fetchErrorData = useSelector(selectFetchNotesError);` line).

That way we have a block of `useSelector` code at the top of the component, then another block of code to calculate the `tabelNotes`, then the rest...?